docs/src/network_analysis/crn_theory.md

```diff
@@ -0,0 +1,332 @@
+> Note, currently API functions for network analysis and conservation law analysis
+> do not work with constant species (currently only generated by SBMLToolkit).
```

Comment:
I'd probably remove this one, doesn't feel like something we need everywhere, given the niche ness of the situation.

If it is something that can yield a silent error, it might make sense to add it as a warning later on. Howeverstarting the tutorial with a note about something that most users won't know about or encounter seems irrelevant. And if it is restricted to SBMLToolkit the warning might be better there.